CSL JSON.js

```diff
@@ -56,6 +56,7 @@ function detectImport() {
 function doImport() {
 	var parsedData = parseInput();
 	if(!parsedData) return;
+	if(!parsedData.length) parsedData = [parsedData];
```

Comment:
That's what I thought -- and it has length 0 in that case? That's what confused me. 
`!parsedData.isArray()` seems clearer/more suitable?